src/amo/components/Addon/index.js

```diff
@@ -552,8 +552,9 @@ export class AddonBase extends React.Component {
 
             {this.renderShowMoreCard()}
 
-            {config.get('enableAddonRecommendations') && (
-              <AddonRecommendations addon={addon} />
+            {config.get('enableAddonRecommendations') &&
+             addonType === ADDON_TYPE_EXTENSION && (
```

Comment:
Can we extract these conditions into a new variable above? `enableAddonsRecommendations`?

Reply:
I can, but I didn't because it's only used once, so I didn't see the point in creating a variable for it. Is it just to improve readability? I'm fine doing it, and I will, but I just want to understand why you think it's important.

Reply:
I think it would be slightly more readable, and if we open the door to other addon types, that will be much better.

Also, I think we had an issue once with multiple conditions chained with `&&` (rendering `0` instead of not rendering anything).

Reply:
Oh yeah, we did. I think that was because one of the conditions returned a `0` instead of a boolean, but yes, this is a good thing to change. 👍